code/components/citizen-resources-gta/src/GameInputFunctions.cpp

```diff
@@ -27,6 +27,17 @@ static InitFunction initFunction([]()
 		}
 	});
 
+	fx::ScriptEngine::RegisterNativeHandler("SET_KEY_MAPPING_HIDE_RESOURCES", [](fx::ScriptContext& context)
+	{
+		bool hide = context.GetArgument<bool>(0);
+
+		fx::OMPtr<IScriptRuntime> runtime;
+		if (FX_SUCCEEDED(fx::GetCurrentScriptRuntime(&runtime)))
+		{
+			game::SetKeyMappingHide(hide);
```

Comment:
Also rename the cpp function not just the native.

Reply:
Should be good